lint/stylelint.bzl

```diff
@@ -0,0 +1,208 @@
+"""Configures [Stylelint](https://stylelint.io/) to run as a Bazel aspect
+
+First, all CSS sources must be the srcs of some Bazel rule.
```

Comment:
Is "some Bazel rule" correct or is it  only css_library rules because of the `rule_kinds = ["css_library"]` default?

Reply:
it's some Bazel rule. `filegroup` is what I imagine everyone has to do today. `css_library` is a forward-looking fiction.

Reply:
ah @dgp1130 made one :)
https://github.com/dgp1130/rules_prerender?tab=readme-ov-file#api

Reply:
Yeah, `css_library` is something I'd very much like to *not* have. I unfortunately had to make something minimal out of necessity since it is "forward-looking fiction" at this point.